packages/@okta/vuepress-site/docs/guides/oin-sso-overview/index.md

```diff
@@ -0,0 +1,73 @@
+| **Resources** | <ul><li>[OpenID Connect Foundation](https://openid.net/connect/)  </li></ul>| <ul><li>[SAML 2.0 Technical Overview](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html) </li></ul> |
+
+## Use case examples
+
+### Example of a workforce user Single Sign-On journey with Okta
+
+1. Ramon is starting his work day. In his web browser, he clicks on the Okta browser extension and selects his email app which loads in a new tab.
+2. Ramon’s company has an Okta sign-on policy, which requires each employee to verify their identity every 12 hours. Since it’s been more than 12 hours since he was last working, he is prompted to enter his Okta username and password.
+3. Ramon’s company has also enabled Okta multifactor authentication, so after successfully entering his credentials, a push notification is sent to the Okta Verify app on his phone. Ramon taps his phone to verify his identity. Ramon can now access his email!
+4. Next, Ramon goes to his Okta browser extension and selects his customer relationship management app. Since Ramon started a session less than 12 hours ago, he is taken directly into the app without needing to sign-in again. In fact, Ramon can access all the Okta-integrated apps that he has privileges to without signing in again because he already has an authenticated session with Okta.
+
+### Example of a partner integration journey with Okta
+
+#### Scenario
+
+Erika is an application developer at Acme, a technology partner with Okta. Acme is looking to leverage the OIN as a way for their customers to adopt and incorporate Acme’s application to the customer’s existing Okta tenant. This will allow Acme’s customers to add Acme’s app to their existing identity infrastructure with minimal configuration. In addition, with a pre-build Acme-Okta integration, Acme will avoid additional integration resources and support for each individual customer.
+
+#### Actions
+
+1. Erika builds the Acme-Okta integration, doing the heavy lifting so that their customers don’t have to.
+2. Erika submits the integration for Okta to verify and review.
+3. After approval, Acme’s app is published on the OIN.
+
+#### Results
+
+The next Acme customer that wants to add the OIN-published Acme app into their own workforce ecosystem, minimal effort is required for the integration. The customer already knows that they can trust the apps in the OIN to be securely incorporated into their existing Okta-managed SSO. The customer’s employees sign in to the Acme app with their existing Okta credentials and no additional Acme app registration is required.
+
+## Next steps
+
+Want to automate even more for your customers and increase adoption of your product? Learn more about [lifecycle management integration](/docs/guides/oin-lifecycle-mgmt-overview/) in the OIN.
```

Comment:
is the next step to start building? Maybe this is explore more or explore other integration types? I think it would be cool to have a link to LCM further up but don't feel too strongly about it.

Reply:
We already have links to "Build an SSO integration" in 2 sections above. Should we assume that their next step is to submit the integration? I can add the Build SSO link again.
Also: I've added a link to view current integrations.

Reply:
OK. My thinking was that this page does not give you information on how to build / implement an integration, so I saw implementing SAML or OIDC as the next step.